onnx/defs/tensor_proto_util.cc

```diff
@@ -60,10 +60,9 @@ namespace ONNX_NAMESPACE {
         }                                                                  \
       }                                                                    \
     }                                                                      \
-    res.insert(                                                            \
-        res.end(),                                                         \
-        reinterpret_cast<const type*>(bytes),                              \
-        reinterpret_cast<const type*>(bytes + raw_data.size()));           \
+    const size_t raw_data_size = raw_data.size();                          \
```

Comment:
Would be good to capture why we do this in a comment so it isn't accidentally removed in the future.

Reply:
@gwang-msft
The culprit is reinterpret_cast, which is a warn sign. 
Your change is good, but the reinterpret_cast is unnecessary.

Reply:
Yes, the reinterpret_cast here is redundant, can remove in a future change